scripts/src/main/resources/scripts/functions

```diff
@@ -711,6 +714,13 @@ function doUnzip() {
   doRunCommand "unzip -qn '${1}' -d '${target_dir}'"
 }
 
+# $1: filename (e.g. ${filename})
+function doIsInstallerExtension()
+{
+  local myext="${1/*\./.}"
+  return $([ "${myext}" = ".msi" ] || [ "${myext}" = ".exe" ] || [ "${myext}" = ".pkg" ] || [ "${myext}" = ".bat" ])
```

Comment:
The syntax `$(...)` is causing that the "command" between the parentheses gets executed.
The expression in the parentheses here will be evaluated to something like `0` (true/success) or e.g. `1` or `255` (false/failure).
Therefore you IMHO have to remove this extra syntax or this can not work as expected.